pxr/imaging/glf/CMakeLists.txt

```diff
@@ -1,87 +1,157 @@
 set(PXR_PREFIX pxr/imaging)
 set(PXR_PACKAGE glf)
-
-if (NOT ${PXR_ENABLE_GL_SUPPORT})
-    message(STATUS
-        "Skipping ${PXR_PACKAGE} because PXR_ENABLE_GL_SUPPORT is OFF")
-    return()
-endif()
+set(optionalLibs "")
+set(optionalIncludeDirs "")
 
 set(optionalPublicClasses "")
 if (X11_FOUND)
     list(APPEND optionalPublicClasses testGLContext)
 endif()
 
-set(optionalLibs "")
-set(optionalIncludeDirs "")
+if (NOT ${PXR_ENABLE_GL_SUPPORT})
+    message(STATUS
```

Comment:
I'm mentally struggling to spot the difference between the two clauses here, and wondering if the difference might be encapsulated in a cmake var instead? and conditionalize the contents of the var?

Reply:
Agree, modified this quite a bit to simplify and moving into variables